metadata-models/src/main/pegasus/com/linkedin/ml/metadata/MLModelProperties.pdl

```diff
@@ -0,0 +1,46 @@
+namespace com.linkedin.ml.metadata
+
+import com.linkedin.common.MLFeatureUrn
+import com.linkedin.common.Time
+import com.linkedin.common.VersionTag
+
+/**
+ * Properties associated with a ML Model
```

Comment:
Another suggestion from the RFC is to link the training pipeline code which is used to train this specific model. Think that will be reasonable?

Reply:
This is similar to the comment of adding execution job/process. Again we can add this optional field later.

Reply:
I like this idea, Will try to add like an optional sourcecode map with enum keys as MODEL_SOURCE_CODE, TRAINING_PIPELINE_SOURCE_CODE, EVALUATION_PIPELINE_SOURCE_CODE??

Reply:
Looks like a promising idea to link all different types of source code. Thanks for the suggestion!
Do you want to capture these source code links as separate fields, perhaps in a separate aspect?

Reply:
Thinking more, you could as well add field for `TRAINING_PIPELINE_SOURCE_CODE` to `TrainingData` aspect, etc. I will let you evaluate different options and decide on one.
Again this is not a blocker, these fields can be added in the next PR as well.

Reply:
Got it. As of now added `ML_MODEL_SOURCE_CODE`, `TRAINING_PIPELINE_SOURCE_CODE` and `EVALUATION_PIPELINE_SOURCE_CODE` source code types to the `SourceCode` aspect.